data/remnant/remnant persons.txt

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+person "Prototype B3-CC4"
+	frequency 20000
+	government "Author"
+	personality
+		forbearing mining surveillance unconstrained
+	phrase
+		phrase
+			"friendly author"
+	system
+		attributes "ember waste"
+	phrase
+		word
+			"I may be just a piece of rock, but"
+			"Even for a floating gathering of debris"
```

Comment:
This line doesn't really make sense to me with the potential follow-ups.
"Even for a floating gathering of debris you shouldn't buy anything the Syndicate sells."
Both these first two lines could use something a little extra to make them work out, really.
```suggestion
			"I may be just a piece of rock, but I know that"
			"Even for a floating gathering of debris I know that"
```